opcua/uatypes.py

```diff
@@ -869,7 +869,7 @@ def from_binary(data):
         return obj
 
     def __str__(self):
-        s = 'DataValue(Value:{})'.format(self.Value)
+        s = 'DataValue(Value:{}'.format(self.Value)
```

Comment:
all good apart from this one. merging anyway. hopefully one of use will remember to fix it back